src/send/saga.ts

```diff
@@ -264,9 +284,17 @@ export function* sendPaymentSaga({
 
     yield* put(sendPaymentSuccess({ amount, tokenId }))
     SentryTransactionHub.finishTransaction(SentryTransaction.send_payment)
-  } catch (e) {
-    yield* put(showErrorOrFallback(e, ErrorMessages.SEND_PAYMENT_FAILED))
+  } catch (err) {
+    yield* put(showErrorOrFallback(err, ErrorMessages.SEND_PAYMENT_FAILED))
     yield* put(sendPaymentFailure())
+    const error = ensureError(err)
+    if (error.message === ErrorMessages.PIN_INPUT_CANCELED) {
+      Logger.info(`${TAG}/sendPaymentSaga`, 'Send cancelled by user')
+      return
+    }
+    Logger.error(`${TAG}/sendPaymentSaga`, 'Send payment failed', error)
+    ValoraAnalytics.track(SendEvents.send_tx_error, { error: error.message })
+    SentryTransactionHub.finishTransaction(SentryTransaction.send_payment)
```

Comment:
Should the sentry transaction be in a `finally` block?

Reply:
This was my original plan, but if the pin is cancelled, we don't start the transaction, so I included it here. But looking at finishTransaction, it looks like it does nothing if a tx with given name isn't found, so should be safe and clean to do it in finally.